test/sharness/t0060-daemon.sh

```diff
@@ -8,33 +8,31 @@ test_description="Test daemon command"
 
 . lib/test-lib.sh
 
-gwyport=8080
-apiport=5001
+gwyport=8082
+apiport=5002
+swarmport=4003
+
+API_MADDR="/ip4/0.0.0.0/tcp/$apiport"
```

Comment:
now that youre using `test_launch_ipfs_daemon` it will handle setting these `_MADDR` variables for you:

https://github.com/ipfs/go-ipfs/blob/master/test/sharness/lib/test-lib.sh#L196